src/main/java/core/basesyntax/model/StorageTransaction.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+import java.util.Arrays;
+
+public class StorageTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public StorageTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private String code;
+
+        Operation(String code) {
+            this.code = code;
+        }
+
+        public static Operation getOperation(String code) {
+            return Arrays.stream(Operation.values())
+                    .filter(t -> t.getCode().equals(code))
+                    .findFirst()
+                    .get();
```

Comment:
What happens if you try to get an operation, that you don't have yet? For example typo mistake in the input file?